tests/jenkins/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM jenkins/jenkins:lts
+#FROM jenkins/jenkins:lts
```

Comment:
Did you mean to comment out this line?

If so, you should probably just remove it -- no need to commit stale/commented out code.